api/src/main/java/org/openmrs/module/patientqueueing/api/dao/PatientQueueingDao.java

```diff
@@ -28,4 +38,71 @@ public DbSession getSession() {
 		return sessionFactory.getCurrentSession();
 	}
 	
+	/**
+	 * Get a single patient queue record by parameter
```

Comment:
I thought you would put all these javadoc on the service interface and then here you simply use the `@see` annotatio.